src/index.html

```diff
@@ -11,6 +11,14 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-bar search-bar--big" data-qa="big">
+      <input type="text" placeholder="Try &quot;Los Angeles&quot;"
+             class="search-input search-input--big" data-qa="keypress">
```

Comment:
Fix code style everywhere
```suggestion
      <input type="text" placeholder="Try &quot;Los Angeles&quot;"
             class="search-input search-input--big" data-qa="keypress">
```
```suggestion
      <input
        type="text"
        placeholder="Try &quot;Los Angeles&quot;"
        class="search-input search-input--big"
        data-qa="keypress"
      >
```